Mathlib/Algebra/Order/Hom/Monoid.lean

```diff
@@ -590,6 +590,32 @@ instance : Inhabited (α ≃*o α) :=
 
 variable {α}
 
+/-- Symmetry of multiplication-preserving order isomorphisms -/
+@[to_additive (attr := symm) "Symmetry of addition-preserving order isomorphisms"]
+def symm (f : α ≃*o β) : β ≃*o α where
```

Comment:
Surprising that we didn't have this before but it seems to indeed have been the case!

Reply:
This ended up being added independently in #22402